src/structures/RustPlus.js

```diff
@@ -124,6 +124,25 @@ class RustPlus extends RP {
         }
     }
 
+    handleGame2DiscordChat(message) {
+        let instance = Client.client.readInstanceFile(this.guildId);
+        let channel = DiscordTools.getTextChannelById(this.guildId, instance.channelId.ingamechat);
+        if (message.message.startsWith("rustPlusPlus") || message.message.startsWith("Discord -​> Game")) return;
+        if (channel !== undefined) {
+            let embed = new MessageEmbed()
+                .setColor(message.color)
+                .setTitle(message.name)
+                .setAuthor({ name: "Team Chat Message" })
```

Comment:
Not necessary

Reply:
What part of it is unnecessary? The whole embed?

Having an embed makes it look more clean than just sending a message saying "Name: Message"